pages/common/diffoscope.md

```diff
@@ -11,7 +11,7 @@
 
 `diffoscope --no-progress {{path/to/file1}} {{path/to/file2}}`
 
-- Compare two files and write a HTML-report to a file (use `-` for stdout):
+- Compare two files and write an HTML-report to a file (use `-` for stdout):
```

Comment:
I dislike this change. It is wrong IMO and should be left as it was.

Reply:
https://www.gramlee.com/blog/indefinite-articles-a-or-an-before-h-blame-the-confusion-on-the-french/

Reply:
Is that you agreeing with me, from the document, thats a yes

Reply:
This isn't a subjective matter, "an HTML" is the correct use of a/an.

In English, we say "a" if the next syllable is pronounced starting with the sound of a consonant.
We say "an" if the next syllable starts with the sound of a vowel.

For example:
"an item"
"an element"

"a report"
"a record"

However note it's the _sound_, not the _letter_.

For example, "an HTML" because it's pronounced with an "aye" sound at the start.